pkg/operator/sync.go

```diff
@@ -674,3 +706,19 @@ func newTerminationContainers(config *OperatorConfig) []corev1.Container {
 		},
 	}
 }
+
+// forceSyncDeployment uses inputHash map of external dependencies to force new generation of the deployment
+// triggering the Kubernetes rollout as defined when the inputHash changes.
+func forceSyncDeployment(inputHashes map[string]string, deployment *appsv1.Deployment) {
+	for k, v := range inputHashes {
+		annotationKey := fmt.Sprintf("dep-%s", k)
```

Comment:
We should prefix any annotations with an openshift domain prefix.

"x.openshift.io/dep-%s"

Annotation keys without prefix are supposed to be reserved for the user.